Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.11.0-alpine3.7
```

Comment:
Why 1.11.0 and why Alpine?

Reply:
Alpine because it has a small size.
1.11.0 because I just decided to use it. I can use 1.12 if you want.

Reply:
But why use specific version at all? Why not latest?
And why care about Docker image size more that compatibility?

Reply:
Specific version because tests should be stable and shouldn't fail because of updating go version.
Do you really think that there might be compatibility issues?
We are testing PMM API here, not generated client.